packages/cactus-plugin-satp-hermes/src/main/typescript/config-validating-functions/bridges-config-validating-functions/validateBridgesConfig.ts

```diff
@@ -0,0 +1,99 @@
+import { SupportedChain } from "../../core/types";
```

Comment:
I'm not saying this is the wrong way, or that the one above is better. We can leave this as is, provided it works. But please open an issue in Cacti for refactoring this logic

Reply:
@RafaelAPB @brunoffmateus I'd vote  for the build-in language features over the `class-validator` library because it allows the compiler to have an awareness of the result of runtime checks. What I mean is that if you have a function input parameter that you are validating then with class-validator you know in your head that the execution won't get past the validation (it will crash which is the expected behavior) BUT the compiler won't know this so it will still treat a potentially falsy input parameter as potentially falsy, whereas if you validate with a type-guard (the built-in language feature) then the compiler will start treating the potentially falsy variable as definitely-not-falsy which means that you can eliminate unsafe looking casts from your code (which are a code smell)